### DISCUSSION TITLE
Please create a PR for Display support in Klipper3d/klipper

### DISCUSSION THREAD

**Discussion by LaggAt** [Desuuuu/klipper#123] [General]
Hi, there is a request on the klipper repo:
https://github.com/Klipper3d/klipper/issues/5004

Is it possible to start a PR for T5UID1 touchscreen support there?
Thanks for looking into it.

**Comment by hakuyouhiro** [comment]
Are we covering this at https://klipper.discourse.group/c/features/9 ? i can't see the topic created

**Comment by JaxCoder** [comment]
Looks like the most recent discussion about it is happening here: https://klipper.discourse.group/t/feature-request-extend-protocol-with-uart-commands/2414
